infrastructure/monitoring/kibana/config.ndjson

```diff
@@ -1,7 +1,7 @@
+{"attributes":{"actions":[{"actionRef":"preconfigured:preconfigured-alert-history-es-index","actionTypeId":".index","group":"logs.threshold.fired","params":{"documents":["{\"@timestamp\":\"2023-11-22T08:25:47.329Z\",\"tags\":\"{{rule.tags}}\",\"rule\":{\"id\":\"{{rule.id}}\",\"name\":\"{{rule.name}}\",\"params\":{\"{{rule__type}}\":\"{{params}}\"},\"space\":\"{{rule.spaceId}}\",\"type\":\"{{rule.type}}\"},\"kibana\":{\"alert\":{\"id\":\"{{alert.id}}\",\"context\":{\"{{rule__type}}\":\"{{context}}\"},\"actionGroup\":\"{{alert.actionGroup}}\",\"actionGroupName\":\"{{alert.actionGroupName}}\"}},\"event\":{\"kind\":\"alert\"}}"]}}],"alertTypeId":"logs.alert.document.count","apiKey":null,"apiKeyOwner":null,"consumer":"alerts","createdAt":"2023-11-22T08:32:38.272Z","createdBy":"elastic","enabled":false,"executionStatus":{"error":null,"lastExecutionDate":"2024-02-07T08:28:08.400Z","status":"pending"},"legacyId":null,"meta":{"versionApiKeyLastmodified":"7.17.0"},"muteAll":false,"mutedInstanceIds":[],"name":"Error in backup logs","notifyWhen":"onActionGroupChange","params":{"count":{"comparator":"more than or equals","value":1},"criteria":[{"comparator":"equals","field":"log.file.path","value":"/var/log/opencrvs-backup.error.log"}],"timeSize":1,"timeUnit":"h"},"schedule":{"interval":"1h"},"scheduledTaskId":null,"tags":[],"throttle":null,"updatedAt":"2024-02-07T02:27:22.558Z","updatedBy":"elastic"},"coreMigrationVersion":"7.17.0","id":"b166fcb0-8911-11ee-8111-2f3be9e93efc","migrationVersion":{"alert":"7.16.0"},"references":[],"sort":[1707294436464,232766],"type":"alert","updated_at":"2024-02-07T08:27:16.464Z","version":"WzQ5NTQzNCwxOV0="}
```

Comment:
To be 100% sure no errors are ending up in `/var/log/opencrvs-backup.log` anymore, we'd need to test the old error scenarios and verify the indeed end up in the right file. In the backup script, especially [these can be a little dangerous](https://github.com/opencrvs/opencrvs-countryconfig/blob/d4d8be291bc5145ccd21b118af13a7d39da70cd2/infrastructure/backups/backup.sh#L201-L214) and might not correctly output their error. 

Question you can think about is:
How could we test the different error scenarios are now properly sending an alert and write errors to the `opencrvs-backup.error.log`?

As testing this can be quite time consuming, my suggestion is: let's also keep the old alert rule in place that tries to find the word "error" in the `/var/log/opencrvs-backup.log`. We can then at some point come back to this task and see what type of stuff each file contains and remove this "extra" error rule.

Reply:
If you end up using a different name for the "deprecated" (so the old) alert rule, please note that you need to add a new filter for Elastalert too

https://github.com/opencrvs/opencrvs-countryconfig/blob/242878c13e0ce335c096eed10c59f77be812f805/infrastructure/monitoring/elastalert/rules/log-alert.yaml#L24-L30

Elastalert is a service that 
1. Listens Elasticsearch indices for the alerts triggered 
2. Sends a HTTP request to our country config package's `POST /email` endpoint
3. Country config then sends an email (which is in our environments configured to be a Slack email inbox)

Reply:
Actually I like this idea to keep the old alert rule, that will ensure us about the correctness of our work without taking risk.